README.adoc

```diff
@@ -1,25 +0,0 @@
-= Liquid Crystal Library for Arduino =
```

Comment:
Is there a reason we are switching from asciidoc to markdown?

I believe @mbanzi prefers asscidoc :) cc/ @dcuartielles